src/helpers/layerHelper.ts

```diff
@@ -0,0 +1,12 @@
+import type { VectorLayer } from '@vcmap/core'
+
+export function getFeatureByAttribute(
+  attribute: string,
+  value: string,
+  layer: VectorLayer
+) {
+  const feature = layer.getFeatures().find((feature) => {
+    return feature.getProperty(attribute) === value
```

Comment:
This is not a request for change here just a remark
I once had a problem using a layer on which the data was not yet loaded
To remedy the problem I had to add the line below to wait for the layer data to be loaded:
`await layer.fetchData();`

I don't know if this line is needed here but at least you have the code if you ever need it

Reply:
Thanks, @toni-garcia-freelance 
I will keep it in my mind if I got the same issue

Reply:
ah, I found it to be necessary when I open the line view directly. I created another PR to add the fetch data: https://github.com/sigrennesmetropole/cooperation_jn_app/pull/25